hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/record/entity/EntityProperties.java

```diff
@@ -55,5 +62,7 @@
          * If configured the mirror node will store the raw transaction bytes on the transaction table
          */
         private boolean transactionBytes = false;
+
+        private Collection<TransactionTypeEnum> transactionSignatures = EnumSet.of(SCHEDULECREATE, SCHEDULESIGN);
```

Comment:
Maybe we should make this a map instead.
```suggestion
        private Map<TransactionTypeEnum, boolean> transactionSignatures = Map.of(SCHEDULECREATE, true SCHEDULESIGN, true);
```

This would avoid the list search noted below